File: easterEggs/CHINCHIN/Lottery.java
import java.security.SecureRandom;
import java.util.Scanner;


public class Lottery{
public static void main(String[] args){
        
        Scanner sc = new Scanner(System.in);
        SecureRandom random = new SecureRandom();


int gen = random.nextInt(90)+10;

System.out.print("Enter a two digit Number:");
int number = sc.nextInt();

int genDigit1= gen /10;
int genDigit2= gen % 10;

int numDigit1= number /10;
int numDigit2= number % 10




if (number == gen){

    System.out.print("You win $10,100");


}  else if (    (numDigit1==genDigit1 || numDigit1 == genDigit2) && (numDigit2==genDigit1 || numDigit2 == genDigit2)    ){

        System.out.print("You win $3000");

} else if (    (numDigit1==genDigit1 || numDigit1 == genDigit2) || (numDigit2==genDigit1 || numDigit2 == genDigit2)    ){

        System.out.print("You win $1000");

} 






}

}

File: Python eastereggs/chinchin/lottery.py
import random




gen = random.range(90)+10


number = int(input("Enter a two digit Number:"))

genDigit1= gen /10
genDigit2= gen % 10

numDigit1= number /10
numDigit2= number % 10




if number == gen:

    print("You win $10,100")


if (    (numDigit1==genDigit1 or numDigit1 == genDigit2) and (numDigit2==genDigit1 or numDigit2 == genDigit2) :

        print("You win $3000")

if (    (numDigit1==genDigit1 or numDigit1 == genDigit2) or (numDigit2==genDigit1 or numDigit2 == genDigit2):

        print("You win $1000")


